unit_test.sh

```diff
@@ -23,7 +23,7 @@ usql $TEST_DB_URL -f db/test_synthetic_beneficiaries.sql
 usql $TEST_DB_URL -f db/worker.sql
 
 echo "Running unit tests and placing results/coverage in test_results/${timestamp} on host..."
-DATABASE_URL=$TEST_DB_URL QUEUE_DATABASE_URL=$TEST_DB_URL gotestsum --junitfile test_results/${timestamp}/junit.xml -- -race ./... -coverprofile test_results/${timestamp}/testcoverage.out 2>&1 | tee test_results/${timestamp}/testresults.out
+DATABASE_URL=$TEST_DB_URL QUEUE_DATABASE_URL=$TEST_DB_URL gotestsum --format standard-verbose --junitfile test_results/${timestamp}/junit.xml -- -race ./... -coverprofile test_results/${timestamp}/testcoverage.out 2>&1 | tee test_results/${timestamp}/testresults.out
```

Comment:
Wasn't there some reason for using the default format because it didn't clutter the Jenkins/Travis output?

Reply:
this was a mistake to commit.  I will fix it.